docs/upgrade/UPGRADE-v7.0.0-beta6.md

```diff
@@ -159,7 +167,6 @@ for instance:
             - use new methods `getAllEnabledOnDomain` and `getAllEnabledOnCurrentDomain` (methods returns only enabled countries)
             - change usages in `BillingAddressFormType`, `DeliveryAddressFormType` and `PersonalInfoFormType` in your implementation
             - fix `CountryFacade` mock in `PersonalInfoFormTypeTest` – mock method `getAllEnabledOnDomain` instead of `getAllByDomainId`
```

Comment:
```suggestion
            - fix `CountryFacade` mock in `PersonalInfoFormTypeTest` – mock method `getAllEnabledOnDomain` instead of `getAllByDomainId`
            - fix all usages of method `Country::getName` if exist, as it now needs proper locale as an argument
```